testing/run_tests.py

```diff
@@ -1053,7 +1053,7 @@ def __exit__(self, exception_type, exception_value, exception_traceback):
     os.chdir(self.old_cwd)
 
 
-def run_impeller_golden_tests(build_dir: str):
+def run_impeller_golden_tests(build_dir: str, require_skia_gold: bool = False):
```

Comment:
We shouldn't need this flag because run_tests.py already has this flag in the form of "--type=impeller-golden", right?

Reply:
There's no reason to run the impeller-golden type when require_skia_gold is false?

Reply:
There is, as-per the PR title - to check if the tests run without validation errors?

Reply:
ahh yea, validation errors